<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clientmanagement</class>
 <widget class="QWidget" name="clientmanagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1221</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font-family:century gothic;
font-size:17px;
background:#333;
}


QPushButton
{
background:blue;
border-radius:15px;
}
QToolButton
{
background:blue;
border-radius:60px;
}
QTextEdit
{
bakcground:#333;
border-radius:60px;
}
QLabel
{
color:white;
}
QWidget
{
color:white;
}
QPushButton:hover
{
background:grey;
}
QLineEdit
{
border:none;
color:white;
border-bottom:1px solid #717072;
}
QTableView
{
background:#333;
}
QHeaderView {
background:#333;
background-color:transparent;
}
QToolButton
{
background:#333;
border-radius:60px;
}
</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1221</width>
     <height>791</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QTableView" name="clientTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>11</y>
       <width>611</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="browseBtn">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>440</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addClient">
     <property name="geometry">
      <rect>
       <x>701</x>
       <y>240</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#addClient
{
	qproperty-icon: url(18/plus-18.png);
	qproperty-iconSize: 18px 18px;
	text-align: center;
}</string>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="searchBox">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>320</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Search here...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pdfClient">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>470</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#pdfClient
{
	qproperty-icon: url(ressources/images/pdfIcon.png);
	qproperty-iconSize: 20px 20px;
	text-align: center;
}</string>
     </property>
     <property name="text">
      <string>Generate PDF</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="subject">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>400</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Subject</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="msg">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>470</y>
       <width>401</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="rcpt">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>360</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Recipiant</string>
     </property>
    </widget>
    <widget class="QLabel" name="count">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>270</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#count
{
	color: rgb(87, 43, 198)
}</string>
     </property>
     <property name="text">
      <string>RowCount</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>360</y>
       <width>251</width>
       <height>95</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Order by :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="sort">
          <item>
           <property name="text">
            <string>ID</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Last Name</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>First Name</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Phone Number</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>E-Mail</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="ascClient">
          <property name="text">
           <string>Asc</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="descClient">
          <property name="text">
           <string>Desc</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="delClient">
     <property name="geometry">
      <rect>
       <x>911</x>
       <y>240</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#delClient
{
	qproperty-icon: url(18/cancel-18.png);
	qproperty-iconSize: 18px 18px;
	text-align: center;
}</string>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QPushButton" name="editClient">
     <property name="geometry">
      <rect>
       <x>806</x>
       <y>240</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#editClient
{
	qproperty-icon: url(18/edit-18.png);
	qproperty-iconSize: 18px 18px;
	text-align: center;
}</string>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="file">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>440</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>File</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>20</y>
       <width>321</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="fn">
          <property name="text">
           <string>First Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="fnEdit">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>John</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="ln">
          <property name="text">
           <string>Last Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lnEdit">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>Smith</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="id">
          <property name="text">
           <string>ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="idEdit">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>12345678</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="ph">
          <property name="text">
           <string>Phone Number</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="phEdit">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>12345678</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="m">
          <property name="text">
           <string>E-Mail</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="mEdit">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>john.smith@esprit.tn</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="sendBtn">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>670</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mail_pass">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>360</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Azizimen1</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>320</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Check Code</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
